docs/documentation/getting-started.md

```diff
@@ -59,29 +60,52 @@ On Windows, you can either use Intel Compilers with the standard Microsoft toolc
+   <summary><h3>Windows + WSL (Recommended)</h3></summary>
 
-Then, in order to initialize your development environment, open a terminal window and run:
+Install [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/) on Windows 11:
+Either
+1. Open a terminal with administrator privileges and run the following command:
 ```shell
-"C:\Program Files (x86)\Intel\oneAPI\setvars.bat"
+wsl --install
 ```
+Or
+1. Open the Start menu, search for "Windows Features", and select "Turn Windows features on or off". Enable "Windows Subsystem for Linux" by checking the corresponding box.
+2. Open the Microsoft Store, search for "Linux", and install your preferred distribution (e.g., [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV))
 
-To follow this guide, please replace `./mfc.sh` with `mfc.bat` when running any commands. `./mfc.sh` is intended Unix-like systems.
-You will also have access to the `.sln` Microsoft Visual Studio solution files for an IDE (Integrated Development Environment).
+Useful software to install for using WSL on Windows:
+- [Windows Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701)
+- [Visual Studio Code](https://code.visualstudio.com/) and the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) extension
+
+Once you have WSL installed, you can follow the instructions for *nix systems above (for Ubuntu, see [Via Aptitude](#via-aptitude)).
 
   </details>
 
   <details>
-     <summary><h3>Windows + WSL</h3></summary>
 
-Install the latest version of the [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/) as well as a distribution such as Ubuntu which can be found [here](https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV). Acquiring an   interactive session is as simple as typing `wsl` in your command prompt, or alternatively, selecting the distribution from the dropdown menu available in the [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701).
+   <summary><h3>Native Windows (Intel)</h3></summary>
+
+Install the latest version of:
+- [Microsoft Visual Studio Community](https://visualstudio.microsoft.com/)
+- [Intel® oneAPI Base Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html)
+- [Intel® oneAPI HPC Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/hpc-toolkit-download.html)
+- [Strawberry Perl](https://strawberryperl.com/) (Install and add `C:\strawberry\perl\bin\perl.exe` or your installation path to your [PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/))
+Please note that Visual Studio must be installed first, and the oneAPI Toolkits need to be configured with the installed Visual Studio, even if you plan to use a different IDE.
```

Comment:
@ChrisZYJ Could you explain what you mean by this?
> the oneAPI Toolkits need to be configured with the installed Visual Studio

This might be what I was missing!

Reply:
It's this step:
![image](https://github.com/user-attachments/assets/e00a77a3-7706-4d82-94bf-507e06acdf31)

I heard it wouldn't work without this, but not very sure

Reply:
I see what you mean, that makes sense. I also did this.

Reply:
For me I had to upgrade both Visual Studio and oneAPI to their 2024 versions before CMake managed to find the Fortran compiler

Reply:
That gives me hope! I am now updating my VS 2022.